src/poetry/vcs/git/backend.py

```diff
@@ -231,11 +231,8 @@ def _clone_legacy(url: str, refspec: GitRefSpec, target: Path) -> Repo:
 
         try:
             SystemGit.clone(url, target)
-        except CalledProcessError:
-            raise PoetryConsoleError(
-                f"Failed to clone {url}, check your git configuration and permissions"
-                " for this repository."
-            )
+        except CalledProcessError as e:
+            raise PoetryConsoleError(f"Failed to clone {url}\n {e.stderr}")
```

Comment:
I would suggest the git error be surfaced via `logger.debug` instead. This is too noisy for 

```suggestion
            logger.debug("Git command returned the following error:\n%s", e.stderr)
            raise PoetryConsoleError(
                f"Failed to clone {url}, check your git configuration and permissions"
                " for this repository."
            )
```